### ISSUE TITLE
Error on all components with useOverlay 

### ISSUE THREAD

**Issue by budarin** [adobe-private/react-spectrum-v3#550]
When try to open an overlay - the error occures
```
react-dom.production.min.js:120 Uncaught TypeError: Cannot read property 'current' of undefined
    at useDialog.ts:36
    at Gi (VM95 client.f11ff4f5.js:31)
    at uu (VM95 client.f11ff4f5.js:31)
    at r.unstable_runWithPriority (VM95 client.f11ff4f5.js:40)
    at Da (VM95 client.f11ff4f5.js:31)
    at lu (VM95 client.f11ff4f5.js:31)
    at Hl (VM95 client.f11ff4f5.js:31)
    at gr (VM95 client.f11ff4f5.js:31)
```

**Comment by devongovett** [comment]
Yeah I think we updated the API and haven't fixed the docs yet. Will address soon.

**Comment by devongovett** [comment]
Fixed in #591.
